lib/portage/dbapi/bintree.py

```diff
@@ -964,11 +964,17 @@ def _populate_remote(self, getbinpkg_refresh=True):
 						download_timestamp + ttl > time.time():
 						raise UseCachedCopyOfRemoteIndex()
 
+				# Set proxy settings for _urlopen -> urllib_request
+				for setting in ('http_proxy', 'https_proxy'):
+					proxies = {}
+					if self.settings.get(setting, None):
+						proxies[setting.replace('_proxy', '')] = self.settings.get(setting)
```

Comment:
I think it will be more straightforward to:

```suggestion
				for proto in ('http', 'https'):
					proxies = {}
					value = self.settings.get(proto + '_proxy')
					if value is not None:
						proxies[proto] = value
```

I suppose this could also use list comprehension.